src/app/core/footer/footer.component.css

```diff
@@ -0,0 +1,3 @@
+mat-toolbar-row {
+  justify-content: space-between;
+}
```

Comment:
Since we need only one CSS style, we can add it directly to the `styles` property of the component decorator.